vsintegration/src/FSharp.Editor/Navigation/GoToDefinitionService.fs

```diff
@@ -336,7 +336,10 @@ type internal FSharpGoToDefinitionService [<ImportingConstructor>]
                         return results.AsEnumerable()
                     else // we need to get an FSharpSymbol from the targetRange found in the signature
                          // that symbol will be used to find the destination in the corresponding implementation file
-                        let implFilePath = Path.ChangeExtension (sigDocument.FilePath,"fs")
+                        let implFilePath =
+                            // Bugfix: apparently sigDocument not always is a signature file
+                            if isSignatureFile sigDocument.FilePath then Path.ChangeExtension (sigDocument.FilePath, "fs") 
+                            else sigDocument.FilePath
```

Comment:
Is this really the right place to do this? shouldn't it happen around ln 297?

Scripts probably need their own full branch of logic to handle [jumping across load directives](https://github.com/fsprojects/VisualFSharpPowerTools/blob/master/src/FSharp.Editing.VisualStudio/Navigation/GoToDefinitionFilter.fs#L72-L75) and making sure they end up at the right destination within a project (imho it should be in the impl file not the sig)